.buildkite/pipeline.integration-test-matrix.yml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
```

Comment:
What is the purpose of this environment variable? I cannot find any references to it in this PR.